### DISCUSSION TITLE
Meeting Body Location Address Display Online (TC) v Online

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#413] [Q&A] [answered]
For online meetings with TC type, in v3.10.0, the address was dark grey and text-decoration: line-through. 

For online meetings without TC type, in v3.10.0, the address was black and no strike-through.

However, in v3.11.0, both inactive and online are the same which is strike-through. This should not be the case for online that never met at a location. I would like to modify the CSS so that these kind of meetings do not have the address with a strike-through.

Are you no longer making a distinction between online and TC type meetings to just online meetings?

**Comment by tech2serve** [comment]
You should feel free to modify it as you see fit for your site. 

If a meeting cannot be attended in-person, then I believe any specific address should be lined out. We don’t want to accidentally send a person to an address that is simply there because a meeting is temporarily inactive.

We’ll continue to evaluate how we can simplify the UI. Less is more.

**Reply by Grateful-Viv** [reply]
The “specific” address is the county and state. There is no specific address to send someone to. It’s not listed as TC. I downloaded the csv and that row doesn’t contain that. How can I exclude these meetings from the rule being used for online AND TC?

**Reply by tech2serve** [reply]
Post a screen cap please.

**Comment by Grateful-Viv** [comment]
Below is online (not TC) in v3.10.0
![Daily-Reflections-310](https://user-images.githubusercontent.com/62247421/124649249-3dcfef80-de66-11eb-9443-be3644f13625.png)

Below is same meeting in v3.11.0
![Daily-Reflections](https://user-images.githubusercontent.com/62247421/124649291-4cb6a200-de66-11eb-81c1-d0848700503c.png)

If I look at CSV files, neither of these are listed as Location Temporarily Closed. The download from 3.11.0 now has online listed in new attendance option field.

**Reply by tech2serve** [reply]
We'll take a look at that and discuss it.

**Comment by Grateful-Viv** [comment]
Right now you have 4 types of Attendance Options: hybrid, online, in-person & inactive. 

Is it possible to add a Attendance Options type for online only meetings that never met inside? They do not have TC status. Perhaps call it online-only, or rename online to online-tc, and then use online for these types of meetings. You do not have to list it in the drop-down in the same way you do not list hybrid meetings. 

That way, the CSS can be applied to not cross out the address for these particular meetings.

Also, this should still be an open issue. You can call it a feature request instead of bug. 

Thank you.

**Reply by tech2serve** [reply]
The way we look at this is online means a user can attend online. It is automatically (if you will) online only if it isn't also in-person.

If a meeting can be attended online _and_ in person, we include both flags for those who want that distinction.

If a meeting is online and TC, the "Temporarily Inactive" flag is not applied as the meeting is only available online.

BTW, county (or city) and state are approximate addresses and not specific. 

But I do view the fact that an approximate address is lined out doesn't make a lot of sense to me, but I want to talk with the rest of the team to see if there is a reason for this I'm missing.  I also think we need to look closer at the single meeting page to see if there are other things that should be removed.

**Reply by Grateful-Viv** [reply]
>I also think we need to look closer at the single meeting page to see if there are other things that should be removed.

I'd love to be part of this discussion.

**Reply by Grateful-Viv** [reply]
With this latest update (3.12), I’m not seeing any meetings having the location crossed out if it’s online. Was this intentional? I believe it should be crossed out and if online can’t be separated from online and TC, then cross both out.

**Reply by tech2serve** [reply]
No. It appears I didn't get it right. 

A natively online meeting should still show the general location and not be crossed out.

An online meeting in lieu of a in person meeting, for which the location is temporarily inactive, should have a crossed out address. The maps is correctly not showing, but the address should also reflect the line through style.

I'll try to fix that this afternoon.

**Comment by joshreisner** [comment, marked as answer]
Was this fixed in `3.11.3` @Grateful-Viv @tech2serve? If not, what's still left to do?

**Reply by tech2serve** [reply]
It is good.

From our internal procedure perspective, I had hoped that my adding “Answers #413” in the PR might label this Discussion as such, but no such luck. :-)

**Reply by Grateful-Viv** [reply]
See my response  to #443.

**Reply by tech2serve** [reply]
On my dev site, the behavior is correct:

1. For natively online meetings, in which the address is correctly entered as an approximate (i.e., city and state) the address _is not_ lined out.
2. For online meetings in which the address is specific, but for which the admin has indicated the meeting is not available in person, the specific address _is_ lined out.

If you're not seeing this behavior, please check other styling changes you may have made.

**Reply by Grateful-Viv** [reply]
Sorry, my mistake. It’s working as expected. And thank you.
